qa/TL0_videoreader_test/test.sh

```diff
@@ -29,6 +29,12 @@ do_once() {
 
   # generate file_list.txt from video_files directory
   ls -d video_files/*  | tr " " "\n" | awk '{print $0, NR;}' > file_list.txt
+
+  # generate file_list.txt for label testing
+  echo "${DALI_EXTRA_PATH}/db/video/frame_num_timestamp/test.mp4 0 0 99
+  ${DALI_EXTRA_PATH}/db/video/frame_num_timestamp/test.mp4 1 100 199
+  ${DALI_EXTRA_PATH}/db/video/frame_num_timestamp/test.mp4 2 200 256
+  " > ${DALI_EXTRA_PATH}/db/video/frame_num_timestamp/file_list.txt
```

Comment:
I wonder if we want to output this into   ${DALI_EXTRA_PATH}. Maybe /tmp/ is better?